docs/src/main/asciidoc/security-jwt.adoc

```diff
@@ -7,20 +7,23 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Using JWT RBAC
 include::_attributes.adoc[]
 :categories: security
-:summary: This guide explains how your application can utilize SmallRye JWT to provide secured access to the Jakarta REST endpoints.
+:summary: This guide explains how your application can use SmallRye JWT to provide secured access to the Jakarta REST endpoints.
 :extension-name: SmallRye JWT
 :mp-jwt: MicroProfile JWT RBAC
 :topics: security,jwt
 :extensions: io.quarkus:quarkus-smallrye-jwt
 
-This guide explains how your Quarkus application can utilize https://github.com/smallrye/smallrye-jwt/[SmallRye JWT]
-to verify https://tools.ietf.org/html/rfc7519[JSON Web Token]s, represent them as MicroProfile JWT `org.eclipse.microprofile.jwt.JsonWebToken`
-and provide secured access to the Quarkus HTTP endpoints using Bearer Token Authorization and https://en.wikipedia.org/wiki/Role-based_access_control[Role-Based Access Control].
+This guide explains how to integrate https://github.com/smallrye/smallrye-jwt/[SmallRye JWT] into your Quarkus application to implement JSON Web Token (JWT) security in compliance with the MicroProfile JWT specification (RFC 7519).
```

Comment:
Sorry @rolfedh, Mp JWT spec is essentially an opinionated treatment of RFC7519, where every claim is optional, MP JWT uses this RFC as an input for its work... So I'd drop a reference to RFC because it is not MP JWT. We should probably retain a reference to this RFC when introducing a `JSON Web Token (JWT)` . May be

```
in compliance with the MicroProfile JWT specification which is based on https://tools.ietf.org/html/rfc7519[RFC 7519].
```  

or lets just keep `JsonWebToken` link to it

Reply:
Thank you for the clarification. I've updated the text accordingly.